Server/Routes/contentRoutes.js

```diff
@@ -0,0 +1,16 @@
+const {
+    createContent,
+    getContents, getContent,
+    updateContent, deleteContent
+} = require('../Controllers/contentController');
+const verifyRole = require('../utils/verifyRole');
+const { contentValidator } = require('../utils/validators/contentValidator');
+const router = require('express').Router();
+
+router.post("/", verifyRole("educator"), contentValidator, createContent);
+router.get("/", verifyRole("educator", "student"), getContents);
+router.get("/:id", verifyRole("educator", "student"), getContent);
+router.put("/:id", verifyRole("educator"), contentValidator, updateContent);
```

Comment:
look we use put to help us update the data(even if i was one record) so there is no use of validating the whole data here